package.json

```diff
@@ -50,7 +50,8 @@
       "examples/csharp/*",
       "examples/go/*",
       ".yalc/*",
-      ".yalc/@*/*"
+      ".yalc/@*/*",
+      "test"
```

Comment:
I believe this will pull in a ton of dependencies for all integration tests. Not necessarily a deal breaker, but would be nice to avoid.

Reply:
Yeah, I can get rid of that 👍